common/changes/@microsoft/gulp-core-build/nickpape-add-missing-dependency_2017-08-11-23-22.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/gulp-core-build",
+      "comment": "Add missing orchestrator dependency.",
```

Comment:
I want it to be published... Right now I can't even build web-build-tools using Rush4 because of this missing dependency